ui/src/transaction.ts

```diff
@@ -278,15 +278,20 @@ export enum PayerType {
   User = "user"
 }
 
-// TODO(sos): update to support orgs
-export const formatPayer = (identity: Identity): object => {
-  return {
-    name: identity.metadata.displayName || identity.metadata.handle,
-    type: PayerType.User,
-    avatarFallback: identity.avatarFallback,
-    imageUrl: identity.metadata.avatarUrl
-  };
-};
+// TODO(sos): coordinate payer shape with proxy/registry
+interface Payer {
+  type: PayerType;
+  name: string;
+  avatarFallback?: EmojiAvatar;
+  imageUrl?: string;
```

Comment:
Instead of maybe fields, we could make the Payers variants with guaranteed fields.

Reply:
The `avatarFallback` can be guaranteed but isn't the imageUrl always a maybe field?